plugins/org.jkiss.dbeaver.ext.generic/plugin.xml

```diff
@@ -123,6 +123,8 @@
                     <property id="supports-indexes" label="Driver supports indexes" description="Driver supports table indexes" type="boolean" required="false" defaultValue="true"/>
                     <property id="supports-table-constraints" label="Driver supports constraints" description="Driver supports table constraints (primary keys)" type="boolean" required="false" defaultValue="true"/>
                     <property id="supports-stored-code" label="Driver supports stored code" description="Whether this driver supports stored code (procedures, functions, packages, etc)" type="boolean" required="false" defaultValue="true"/>
+                    <property id="stores-quoted-case" label="Quoted string case" description="Which letter capitalisation is supported for a quoted strings(mixed/upper/lower)" type="string" required="false" defaultValue=""/>
+                    <property id="stores-unquoted-case" label="Unquoted string case" description="Which letter capitalisation is supported for a unquoted strings(mixed/upper/lower)" type="string" required="false" defaultValue=""/>
```

Comment:
```suggestion
                    <property id="stores-quoted-case" label="Quoted string case" description="Letter capitalization of quoted strings (mixed/upper/lower)" type="string" required="false" defaultValue=""/>
                    <property id="stores-unquoted-case" label="Unquoted string case" description="Letter capitalization of unquoted strings (mixed/upper/lower)" type="string" required="false" defaultValue=""/>
```

Reasoning: 

1. We use NA spelling,
2. We cannot use `a` here,
3. Stylistically, it's better to avoid passive voice here.
4. The proposed version is a bit less verbose